docs/_sidebar.md

```diff
@@ -0,0 +1,2 @@
+-   [Home](/)
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6091129)